### ISSUE TITLE
Playnite Web Plugin failing to load

### ISSUE THREAD

**Issue by bfalala** [andrew-codes/playnite-web#590]
Hi,

The version 9.1.0 of the plugin cannot load successfully. It seems a dependency is missing according to the logs (in French) below:

```
System.IO.FileNotFoundException: Impossible de charger le fichier ou l'assembly 'YamlDotNet, Version=16.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e' ou une de ses dépendances. Le fichier spécifié est introuvable.
Nom de fichier : 'YamlDotNet, Version=16.0.0.0, Culture=neutral, PublicKeyToken=ec19458f3c15af5e'
   à PlayniteWeb.PlayniteWeb..ctor(IPlayniteAPI api)

```

Edit:
after adding manually the required dll, a new error:
```
20-10 19:57:10.001|ERROR|ExtensionFactory:Failed to load plugin: Playnite Web
System.IO.FileNotFoundException: Le fichier 'C:\Users\fbenj\AppData\Local\Playnite\extension.yaml' est introuvable.
Nom de fichier : 'C:\Users\fbenj\AppData\Local\Playnite\extension.yaml'
   à System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   à System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   à System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
   à System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
   à System.IO.File.ReadAllText(String path)
   à PlayniteWeb.PlayniteWeb..ctor(IPlayniteAPI api) dans D:\a\playnite-web\playnite-web\apps\PlayniteWebPlugin\src\PlayniteWeb.cs:ligne 60
```

**Comment by 2600box** [comment]
Can confirm, same issue for me too.

**Comment by andrew-codes** [comment]
Looking into this. Thank you for reporting. I'll post back shortly.

**Comment by andrew-codes** [comment]
An update:
- The new dll dependency is not being bundled with the extension file (.pext)
- The version pulled from the `extension.yaml` file is failing as the YAML file is not bundled with the pext file.

I plan to look at resolving these issues tomorrow morning. In the meantime, I would downgrade to 9.0. 9.1's additional feature is more catered to specific situations that are uncommon. For this reason, there is no loss of functionality by waiting for the next patch release. Thanks again!

**Comment by andrew-codes** [comment]
This has been mislabeled in the commit, so it will not automatically comment when released. The resolution is released in 9.1.1.
